changelogs/fragments/73114-fix-ansible-role-download.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - galaxy - support role artifact download from API response `download_url` location (https://github.com/ansible/ansible/issues/73103).
```

These fragments get combined into a restructured text document, not markdown. Use double backticks for inline code formatting.
```suggestion
  - galaxy - support role artifact download from API response ``download_url`` location (https://github.com/ansible/ansible/issues/73103).
```